CONTRIBUTING.md

```diff
@@ -71,9 +70,8 @@ instead.
 When contributing, be aware of a few things:
 
 - Documentation for Kong Hub listings, which includes all Kong Inc.-published
-  and community-published plugins and integrations, lives in the `app/_hub`
-  and `app/_data/extensions` directories. **Versioning is optional, and thus
-  potentially inconsistent, for this part of the documentation**.
+  and community-published plugins and integrations lives in the `app/_hub`
+  and `app/_data/extensions` directories. **Versioning is optional, and thus potentially inconsistent, for this part of the documentation**.
```

Comment:
What is the purpose of this change?

Reply:
The content of that line was being shifted to the next line, so I shifted it back